cms/envs/test.py

```diff
@@ -239,6 +275,9 @@
     }
 }
 MICROSITE_ROOT_DIR = COMMON_ROOT / 'test' / 'test_microsites'
+MICROSITE_TEST_HOSTNAME = 'testmicrosite.testserver'
+MICROSITE_LOGISTRATION_HOSTNAME = 'logistration.testserver'
+
 FEATURES['USE_MICROSITES'] = True
```

Comment:
Please move this feature flag to the top of the microsite configuration block on line 217